mods/ra/maps/soviet-13a/soviet13a.lua

```diff
@@ -0,0 +1,111 @@
+--[[
+   Copyright (c) The OpenRA Developers and Contributors
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+Jeeps = { Jeep1, Jeep2 }
+JeepWaypoints = { JeepWaypoint1.Location, JeepWaypoint2.Location }
+BridgeDefenders = { BridgeAttack1, BridgeAttack2, BridgeAttack3, BridgeAttack4, BridgeAttack5 }
+RadarSites = { Radar1, Radar2, Radar3, Radar4 }
+StartAttack = { StartAttack1, StartAttack2, StartAttack3, StartAttack4, StartAttack5, StartAttack6, StartAttack7, StartAttack8 }
+ChronoDemolitionTrigger = { CPos.New(35,96), CPos.New(36,96), CPos.New(37,96), CPos.New(37,97), CPos.New(38,97), CPos.New(39,97) }
+
+Start = function()
+	Reinforcements.Reinforce(USSR, { "mcv" }, { MCVEntry.Location, DefaultCameraPosition.Location }, 5)
+
+	Utils.Do(Jeeps, function(jeep)
+		jeep.Patrol(JeepWaypoints, true, 125)
+	end)
+
+	Utils.Do(StartAttack, function(a)
+		IdleHunt(a)
+	end)
+
+	ChronoCam = Actor.Create("camera", true, { Owner = USSR, Location = Chronosphere.Location})
+end
+
+MissionTriggers = function()
+	Trigger.OnAllKilledOrCaptured(RadarSites, function()
+		USSR.MarkCompletedObjective(TakeDownRadar)
+		ChronoshiftAlliedUnits()
+	end)
+
+	Trigger.OnCapture(Chronosphere, function()
+		USSR.MarkCompletedObjective(CaptureChronosphere)
```

Comment:
[There are some cells in the road](https://i.imgur.com/KUr5zLs.png) intended to self-destruct the sphere before that could be an issue. However, it's possible to capture a helipad, pilot a transport around that, and drop off an engineer. Loading grenadiers for the wall is optional since the paratroopers include a pair.

It's not quite a skip since the domes are still needed and the next attack wave will destroy the sphere if a backdoor capture succeeds, but I doubt that order was intended.

Shorter: Yeah, probably.

Reply:
Chronosphere will now detonate on capture if radars aren't destroyed.